src/pages/AppConfigure.vue

```diff
@@ -43,15 +45,21 @@ async function handleSave() {
   await updateSmartContractTransactions(appConfigRequestBody)
   currentConfig = store.getters.appConfigRequestBody
   store.commit('hideLoader')
+  toast.success('App Configuration Updated')
```

Comment:
Prefer sticking to sentence casing for all copy. 

```suggestion
  toast.success('App configuration updated')
```

Reply:
Resolved at https://github.com/arcana-network/developer-dashboard/pull/98/commits/f31fd2cec1e84443f7723a2325854fadb617abfb